pkg/actions/gvr.go

```diff
@@ -20,10 +20,13 @@ import (
 	"k8s.io/client-go/restmapper"
 )
 
+var apiGroupRes []*restmapper.APIGroupResources
```

Comment:
We *may* want to use https://pkg.go.dev/sync?tab=doc#Once for this (to make it a little bit cleaner)

Reply:
Updated to use sync